recipes/fans_dashboard/recipe.yaml

```diff
@@ -0,0 +1,50 @@
+context:
+  version: "0.4.1"
+
+package:
+  name: fans-dashboard
+  version: ${{ version }}
+
+source:
+  git: https://github.com/DataAnalyticsEngineering/FANS.git
+  branch: develop
```

Comment:
Please use a tarball from a specific commit, with a hash. Branches are mutable and this build will be impossible to reproduce.

Also, this one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md